source/dub/generators/build.d

```diff
@@ -387,6 +414,14 @@ class BuildGenerator : ProjectGenerator {
 			const pdbFilename = filenames[0].setExtension(".pdb");
 			if (existsFile(build_path ~ pdbFilename))
 				filenames ~= pdbFilename;
+
+			const importLibraryLocation = filenames[0].setExtension(".lib");
+			if (existsFile(build_path ~ importLibraryLocation))
+				filenames ~= importLibraryLocation;
+
+			const exportFilesLocation = filenames[0].setExtension(".exp");
+			if (existsFile(build_path ~ exportFilesLocation))
+				filenames ~= exportFilesLocation;
```

Comment:
I think this entire conditional block can be removed now that we have the dependencyBinariesToCopy parameter here?

Reply:
![Capture](https://user-images.githubusercontent.com/992755/172472627-d6bb081a-3dbf-473f-96d9-cb5e2a8aa418.PNG)

Double checked, nope required.

Reply:
I don't quite understand, isn't the array from the buildTarget function filled with this information? Is this method called elsewhere? seems like this work is duplicated

Reply:
One works on the dependencies for a package that has already been built, and the other runs after building its package.

While they are very similar, the dependency one does have special behavior associated with "linker" files and for switching dll files to be import libraries on Windows.